### PR TITLE
feat: add prompt and isCustomQuestion attributes to custom-forms

### LINKED ISSUE #6111: Add fields for proposed attendee form costumization
Front-end Issue : fossasia/open-event-frontend#3199

Proposed addition to custom-forms field :

- Question Field : To let organisers add details about the custom question (String)
- IsCustomQuestion : To separate the default fields from these custom fields (Boolean)

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #6111 

#### Changes proposed in this pull request:

- Add prompt attribute to work as field prompts
- Add `isCustomQuestion` attribute to differentiate between custom added fields and the ones which were already there 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by uds5501** [comment]
@shreyanshdwivedi @mrsaicharan1 @prateekj117 Please take a look?

**Comment by uds5501** [comment]
@mrsaicharan1 @CosmicCoder96 @shreyanshdwivedi Please review

File: app/api/schema/custom_forms.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ class Meta:
     is_required = fields.Boolean(default=False)
     is_included = fields.Boolean(default=False)
     is_fixed = fields.Boolean(default=False)
+    is_custom_question = fields.Boolean(default=False, allow_none=False)
+    prompt = fields.Str(required=False, allow_none=True)
```
Comment (shreyanshdwivedi) [inline app/api/schema/custom_forms.py]: @uds5501 are both attributes `required` and `allow_none` required here? I guess allow_none will be enough. What say?

Comment (uds5501) [reply app/api/schema/custom_forms.py]: @shreyanshdwivedi  I think you are right. Will fix this.

Comment (uds5501) [reply app/api/schema/custom_forms.py]: @shreyanshdwivedi take a look now?

File: migrations/versions/f5c28956049b_.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'f5c28956049b'
+down_revision = '43e8c59337ae'
```
Comment (mrsaicharan1) [inline migrations/versions/f5c28956049b_.py]: Change the down_revision to the latest migration

File: migrations/versions/f5c28956049b_.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: f5c28956049b
+Revises: 43e8c59337ae
```
Comment (mrsaicharan1) [inline migrations/versions/f5c28956049b_.py]: Change the migration name here as well

File: migrations/versions/f5c28956049b_.py (hunk 2 above)
Comment (uds5501) [reply migrations/versions/f5c28956049b_.py]: @mrsaicharan1 done!

**Comment by CosmicCoder96** [comment]
@uds5501 We need to discuss this approach in the meeting. I am not sure if this is exactly how we will proceed. Putting it on hold until then.

**Pull request closed without merging**